Tasks/AzureFileCopy/Legacy/AzureFileCopy.ps1

```diff
@@ -0,0 +1,180 @@
+param (
```

Comment:
Why do we need Legacy files? Is the ability to switch between PS3 and Legacy handler available?